### PR TITLE
remove 'gibct_filter_enhancement' flag usage

### PR DESCRIPTION
**Pull request by hughes-dustin**
## Description
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/13363

## Testing done
dev tested

## Acceptance criteria
- [x] The feature flag 'gibct_filter_enhancement' is removed.
- [x] The feature flag is not displayed here: https://[environment]-api.va.gov/flipper/features.

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by mdewey — APPROVED** [review]
LGTM

File: src/applications/gi/components/search/InstitutionSearchForm.jsx [hunk 1]
```diff
@@ -46,20 +45,15 @@ function InstitutionSearchForm({
...
-    ? 'Enter a school, location, or employer name'
-    : 'City, school, or employer';
-
   return (
     <div className="row">
       <div id="institution-search" className={filtersClass}>
         <div className="filters-sidebar-inner vads-u-margin-left--1p5">
           {search.filterOpened && <h1>Filter your search</h1>}
-          <h2>{header}</h2>
+          <h2>''Refine search''</h2>
```
[the remark this answers is not in the document]
Comment (hughes-dustin) [reply src/applications/gi/components/search/InstitutionSearchForm.jsx:53]: Oh no. That's a straight typo.

File: src/applications/gi/components/search/InstitutionFilterForm.jsx [hunk 2]
```diff
@@ -142,11 +141,7 @@ class InstitutionFilterForm extends React.Component {
 
     return (
       <Dropdown
-        label={
-          this.props.gibctFilterEnhancement
-            ? 'Institution categories'
-            : 'Institution type'
-        }
+        label={'Institution categories'}
```
[the remark this answers is not in the document]
Comment (hughes-dustin) [reply src/applications/gi/components/search/InstitutionFilterForm.jsx]: Removed the braces.

File: src/applications/gi/components/search/SearchResultTypeOfInstitutionFilter.jsx [hunk 3]
```diff
@@ -25,11 +24,7 @@ function SearchResultTypeOfInstitutionFilter({
 
   return (
     <RadioButtons
-      label={
-        gibctFilterEnhancement
-          ? 'Select an institution type'
-          : 'Type of institution'
-      }
+      label={'Select an institution type'}
```
[the remark this answers is not in the document]
Comment (hughes-dustin) [reply src/applications/gi/components/search/SearchResultTypeOfInstitutionFilter.jsx]: I've removed the braces.

**Review by micahchiang — APPROVED** [review]
LGTM. Left some additional comments about more braces that might be removable.

**Review by rianfowler — APPROVED** [review]

**Review by sonntag-adam — APPROVED** [review]

**Review by Mottie — APPROVED** [review]

**Pull request merged**